runtime/turing/src/lib.rs

```diff
@@ -721,9 +723,26 @@ parameter_types! {
 	pub const MaxBlockWeight: Weight = MAXIMUM_BLOCK_WEIGHT;
 	pub const MaxWeightPercentage: Perbill = SCHEDULED_TASKS_INITIALIZE_RATIO;
 	pub const SecondsPerBlock: u64 = MILLISECS_PER_BLOCK / 1000;
+	pub const ExecutionWeightFee: Balance = 12;
+}
+
+pub struct DealWithExecutionFees<R>(sp_std::marker::PhantomData<R>);
```

Comment:
gosh, the repeated code kills me b/w the 2 runtimes

Reply:
ya I need to do more work to figure out how import more thing from a common runtime